src/main.rs

```diff
@@ -92,7 +99,7 @@ fn main() -> surf::Result<()> {
         };
         worker_wallet.unlock(None).await?;
 
-        // Move money if wallet does not have enough money
+        // make sure the working-wallet has enough money (for paying fees)
```

Comment:
This is not just for paying fees, but actually so that there are UTXOs to even build upon for Melminter.I think the previous comments are more correct throughout.